src/CharterManager.sol

```diff
@@ -222,6 +235,35 @@ contract CharterManagerImp {
         ManagedGemJoinLike(gemJoin).exit(urp, usr, amt);
     }
 
+    function move(address u, address dst, uint256 rad) external allowed(u) {
+        address urp = proxy[u];
+        require(urp != address(0), "CharterManager/non-existing-urp");
+
+        VatLike(vat).move(urp, dst, rad);
+    }
```

Comment:
I think there is some weird thing about commits here, could you please merge `master` on this one to see if this block disappears?

Reply:
I saw you merged master, thanks.

Reply:
Yw!